lib/analyzer.js

```diff
@@ -351,6 +373,17 @@ Analyzer.prototype._processScript = function _processScript(script, href) {
       }.bind(this));
       this.behaviors = this.behaviors.concat(parsedJs.behaviors);
     }
+    if (!this.parsedScripts[href]) {
```

Comment:
`If !(Object.hasOwnProperty.call(this.parsedScripts, href))`

If `href` is `"constructor"`, we shouldn't go adding properties to the global Object function.